packages/backend/server/server.go

```diff
@@ -78,6 +79,14 @@ func SetupServer() *echo.Echo {
 		logger.Log.Error().Str("Error", err.Error()).Send()
 	}
 
+	_, err = c.AddFunc("* * * * *", func() {
+		limiting.GlobalRateLimiter.CleanupOldSubmissions()
```

Comment:
maybe add a note to the cronjob what it means. every month, week, day, hour and minutes it runs.